external_search.py

```diff
@@ -996,7 +1051,8 @@ def __init__(self):
         fields_by_type = {
             "basic_text": ['summary'],
             'filterable_text': [
-                'title', 'subtitle', 'series', 'classifications.term'
+                'title', 'subtitle', 'series', 'classifications.term',
+                'author',
```

Comment:
We index `author` even though we have all the `contributors`, because it dramatically simplifies the handling of queries like  `nalini singh archangel` or `fallen baldacci`. In those cases we want to do a cross-field search including `title` and `contributors.display_name`. I wasn't able to get cross-field search working between a field in the main document and a field in the subdocument, but we have `author` already available as a field in the main document, and searching on that is a lot faster than going into the subdocuments anyway.